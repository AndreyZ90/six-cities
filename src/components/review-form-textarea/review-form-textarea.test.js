import React from 'react';
import TestRenderer from 'react-test-renderer';

import ReviewFormTextarea from '@/components/review-form-textarea/review-form-textarea';

describe(`ReviewFormTextarea component snapshot`, () => {
  test(`Should correctly render ReviewFormTextarea component`, () => {
    const tree = TestRenderer.create(
        <ReviewFormTextarea text={`Hello world`} onTextChange={() => {}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
