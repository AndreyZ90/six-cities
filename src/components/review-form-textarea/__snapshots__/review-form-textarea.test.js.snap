// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewFormTextarea component snapshot Should correctly render ReviewFormTextarea component 1`] = `
<textarea
  className="reviews__textarea form__textarea"
  id="review"
  name="review"
  onChange={[Function]}
  placeholder="Tell how was your stay, what you like and what can be improved"
  value="Hello world"
/>
`;
